### ISSUE TITLE
Update weboob

### ISSUE THREAD

**Issue by bnjbvr** [cozy/cozy-data-indexer#8]
Hi,
# TL;DR

Please update weboob to a more recent version.
# The Longer One

Weboob got some updates that make it safer. For instance I have been hitting an issue that my bank account couldn't be checked. Digging up further, weboob used to check that the SSL certificate used when connecting to a known website (in this case, my bank website) has the same hash that one which is stored in raw in the weboob module of the bank. That's pretty bad, because it means everytime the bank website updates its SSL certificate (which happened a few days ago, because Heartbleed), you have to rewrite a new version of the weboob module. They apparently removed that restriction in most recent versions of weboob, so it'd be nice if the used version of weboob could be updated too.

Cheers!
